graduation_scheme/Projects.md

```diff
@@ -0,0 +1,67 @@
+## Projects in IDSA Open Source Landscape 
+
+This page provides an overview of all projects that are part of the IDSA Graduation Scheme, with newest projects at the top.
+
+:triangular_flag_on_post: To apply with your project, you may [click here](https://internationaldataspaces.org/open-source-application-form/) to apply via the forms on IDSA Website.
+
+:triangular_flag_on_post: And [here](https://github.com/International-Data-Spaces-Association/idsa/tree/main/graduation_scheme) you can check the IDSA Graduation Scheme.
+
+---
+
+
+![Sandbox](media/IDSA-Sandbox-Logo.png)
```

Comment:
the link should be provided relatively  and start with ./ , here ./media/IDSA-Sandbox-Logo.png

Reply:
Updated this one